client/state/plugins/installed/selectors-ts.ts

```diff
@@ -321,3 +322,89 @@ export const getSiteObjectsWithoutPlugin = createSelector(
 	( state: AppState ) => [ getAllPluginsIndexedByPluginSlug( state ), getSitesItems( state ) ],
 	( state: AppState, siteIds: number[], pluginSlug: string ) => [ pluginSlug, ...siteIds ].join()
 );
+
+export function getStatusForPlugin( state: AppState, siteId: number, pluginId: string ) {
+	if ( typeof state.plugins.installed.status[ siteId ] === 'undefined' ) {
+		return false;
+	}
+	if ( typeof state.plugins.installed.status[ siteId ][ pluginId ] === 'undefined' ) {
+		return false;
+	}
```

Comment:
We could convert this to a single condition with some optional chaining:

```suggestion
	if ( typeof state.plugins.installed.status[ siteId ]?.[ pluginId ] === 'undefined' ) {
		return false;
	}
```

Reply:
Good idea!